contents/blog/experiments.md

```diff
@@ -0,0 +1,94 @@
+
+Now, you can choose two metrics:
+
+1. Number of interactions across all graphs, not just funnels.
+2. Number of interactions for funnels.
+
+Which one do you choose?
+
+Note that you're choosing the total interactions here, not unique interactions. Which means if one person clicks on the funnel three times, that counts as 3 interactions for either metric, as it should.
+
+There's a big problem with metric #1: It's global, and a lot more susceptible to things out of our control. For example, if Trends power users are somehow assigned to the control group, you'll unfortunately have a big skew towards control which has nothing to do with the different funnel layouts.
+
+This is indeed something we noticed: The more specific the metric, the fewer outside factors affect your result. Focusing on local optimisation gives you better local information.
+
+At the same time, you don't want to discard second order effects. What if the horizontal funnel layout prevents users from switching to other graphs? This might increase funnel interactions (local metric #2 increases), but at a severe cost to the global metric (#1).
+
+To solve this problem, we introduced secondary metrics. We encourage making the main metric as the local metric, and then allow the option of having a few secondary metrics. We don't do significance analysis on these secondary metrics, but show the metric values for each variant, so you can ensure that there's no huge drop in global metrics while deciding on results.
+
+Another advantage of local metrics over global is that it can be hard to reliably move global metrics[^2], and thus local metrics allow you to see changes faster, since they're smaller scoped, and thus move quicker.
+
+[^2]: Depends a lot on how big and mature your product is, and how many sub features it has. If there's only one thing the app does, the local metric is the global metric.
+
+## Experiment results aren't set in stone
+
+So, you just finished running the experiment above, and the results are in. Horizontal funnel layout had 1000 interactions, while vertical funnel layout had 1200. The results ended up being significant, with vertical funnel layout being 20% better.
+
+All well and good... except this goes against everything you know about using your own product. You find the vertical view congested, hard to parse, and sort of terrible.
+
+Do you blindly trust the data, or your intuition?
+
+Both have issues. Your intuition might be how you see the world, but not necessarily how people who use the product see the world. At the same time, what if there was a bug in the vertical layout implementation, which counted each interaction twice? Thus, the real number could've been 600, instead of 1200, which massively changes your product direction.
```

Comment:
```suggestion
Both have issues. Your intuition might be how you see the world, but not necessarily how people who use the product see the world. At the same time, what if there was a bug in the vertical layout implementation, which counted each interaction twice? Thus, the real number could've been 600, instead of 1,200, which massively changes your product direction.
```